AnsarIbrahim/To-Do-List
# Pair programming - Task 2

### This PR Contains :
- Test
- Add Test: 
   - Edit function tests.
   - Delete task function test.
   - ClearAll completed function tests.
- Add Jest Frameworks
- Linters Check
- Mock
- Used JSDOM
- Pair Programming
##### To get a local copy up and running:
1. git clone the repo.
2. Checkout to the correct branch (Test-2-Todolist).
3.  Run npm install.
4.  Run the npm test
